chromadb/utils/embedding_functions.py

```diff
@@ -676,8 +680,22 @@ def __call__(self, input: Documents) -> Embeddings:
                 self._api_url, json={"instances": [{"content": text}]}
             ).json()
 
-            if "predictions" in response:
-                embeddings.append(response["predictions"]["embeddings"]["values"])
+            predictions = response.get('predictions')
+            if predictions: # This means that `predictions` of response is not None and empty. When `predictions` of response is not satisfied with these conditions, the else branch will be hit.
```

Comment:
let's add `isinstance(predictions, List)` to this condition and consider anything else as error case with only else branch and just say "something went wrong + response". I don't think we need to branch too deep to handle corner cases of the error handling.

Reply:
Sure! I'll add `isinstance(predictions, List) and predctions` as final condition. This condition is to check predictions is a sequence (or list) and make sure that it is not empty. I think it is more completed than before. I'll revise the else branch as well. Your suggestion is more simple and clear. There is no need to make it complicated.